### PR TITLE
Feature: Enable schema management check via config for JaversMongoRepositories

### PR DESCRIPTION
**Pull request by egasimov**
This PR intended to resolve  [Feature: Enable schema management check via config for JaversMongoRepositories #1260](https://github.com/javers/javers/issues/1260)

### CODE REVIEW

**Review by bartoszwalacik — CHANGES_REQUESTED** [review]
please also add this new property to Spring config, see `JaversSqlAutoConfigurationDefaultPropsTest.groovy`

**Comment by egasimov** [comment]
@bartoszwalacik Thanks for reviewing and sharing comments.
I made following changes and would like you re-review my PR

[Make property schemaManagementEnabled(for MongoRepositories) configurable via Spring](https://github.com/javers/javers/pull/1261/commits/e67a485a6eeceb09b5fda5f0fe239688cad85e59) 

[Make snapshotCollectionName, headCollectionName (for MongoRepositories) configurable via Spring](https://github.com/javers/javers/pull/1261/commits/982e55e348a0849443d734bf3f6cbcdc5ec0fc52)

**Comment by egasimov** [comment]
Hey @bartoszwalacik , Did you have any chance to look at PR ?

File: javers-spring-boot-starter-mongo/src/test/groovy/org/javers/spring/boot/mongo/JaversMongoStarterDefaultsTest.groovy [hunk 1]
```diff
@@ -44,6 +44,9 @@ class JaversMongoStarterDefaultsTest extends BaseSpecification{
         javersProperties.objectAccessHook == "org.javers.spring.mongodb.DBRefUnproxyObjectAccessHook"
        !javersProperties.mongodb
         javersProperties.snapshotsCacheSize == 5000
+        javersProperties.schemaManagementEnabled
```
Comment (bartoszwalacik) [inline javers-spring-boot-starter-mongo/src/test/groovy/org/javers/spring/boot/mongo/JaversMongoStarterDefaultsTest.groovy:47]: thanks for adding ale these two missing props

**Review by bartoszwalacik — APPROVED** [review]
LGTM

**Comment by bartoszwalacik** [comment]
need to push it through CI now

**Comment by egasimov** [comment]
@bartoszwalacik ,
Could you please consider these changes as 6.9.2, rather than 7.*.* ?

Because my current project which requires above feature was using java 11 and < Spring 3.0.0.

**Comment by bartoszwalacik** [comment]
@egasimov ok, 6.10.0 is published to the Central

**Comment by bartoszwalacik** [comment]
merged from here https://github.com/javers/javers/pull/1264

**Pull request closed without merging**